Mailbox/Mailbox.cs

```diff
@@ -1,4 +1,33 @@
-﻿namespace Mailbox
+﻿using System;
+
+namespace Mailbox
 {
 
+    public class Mailbox
+    {
+        public Size Size { get; set; }
+        public (int Row, int Column) Location { get; set; }
+        public Person Owner { get; set; }
```

Comment:
Properties can be make readonly.
```suggestion
        public Person Owner { get; }
```